src/components/05-form/search.njk

```diff
@@ -1,7 +1,29 @@
+<h4>SDS Search Example</h4>
 <div class="sds-search">
   <form class="usa-search usa-search--small">
     <div role="search" class="search">
-      <input class="usa-input" id="search-field-small" type="search" name="search">
+      <label class="usa-sr-only" for="formid">Search Here</label>
```

Comment:
The for attributes needs to target the id of the form element, in this case it is search-field-small. Sorry if my comment wasn't clear before. Each form element will need a unique id to do this properly for the whole page.

Reply:
got it!